packages/terra-switch/src/Switch.module.scss

```diff
@@ -0,0 +1,164 @@
+    width: 3.4285rem;
+  }
+
+  .tray.selected {
+    background-color: var(--terra-switch-tray-selected-background-color, #0079be);
+    border: var(--terra-switch-tray-selected-border, 1px solid #01639e);
+    box-shadow: var(--terra-switch-tray-selected-box-shadow, none);
+
+    &.is-disabled {
+      background-color: var(--terra-switch-tray-selected-disabled-background-color, #434a4d);
+      border: var(--terra-switch-tray-selected-disabled-border, 1px solid #434a4d);
+    }
+
+    &:hover:not(.is-disabled) {
+      background-color: var(--terra-switch-tray-selected-hover-background-color, #0065a3);
+      border: var(--terra-switch-tray-selected-hover-border, 1px solid #01639e);
+      box-shadow: var(--terra-switch-tray-selected-hover-box-shadow, inset 0 1px 5px 0 rgba(0, 0, 0, 0.4));
+    }
+  }
+
+  .tray.unselected {
+    background-color: var(--terra-switch-tray-default-background-color, #dedfe0);
+
+    &.is-disabled {
+      background-color: var(--terra-switch-tray-default-disabled-background-color, #d3d4d5);
+      border: var(--terra-switch-tray-default-disabled-border, 1px solid #434a4d);
+    }
+
+    &:hover:not(.is-disabled) {
+      background-color: var(--terra-switch-tray-default-hover-background-color, #b9bbbc);
+      border: var(--terra-switch-tray-default-hover-border, 1px solid #909496);
+      box-shadow: var(--terra-switch-tray-default-hover-box-shadow, inset 0 1px 5px 0 rgba(0, 0, 0, 0.3));
+    }
+  }
+
+  .tray > [aria-checked='true'] {
+    box-shadow: var(--terra-switch-slider-selected-box-shadow, 0 0 4px 1px rgba(0, 0, 0, 0.3), -1px 0 0 0 #01639e);
+    transform: var(--terra-switch-slider-selected-transform, translateX(0.8rem));
+
+    &:hover:not(.is-disabled) {
+      background-color: var(--terra-switch-slider-selected-hover-background-color, #fff);
+      border: var(--terra-switch-slider-selected-hover-border, 0);
+      box-shadow: var(--terra-switch-slider-selected-hover-box-shadow, 2px 0 2px 0 rgba(0, 0, 0, 0.2), -2px 0 2px 0 rgba(0, 0, 0, 0.2), 
+      -1px 0 0 0 #01639e, 0 0 4px 1px rgba(0, 0, 0, 0.2));
+    }
+
+    &:active:not(.is-disabled) {
+      background-color: var(--terra-switch-slider-selected-active-background-color, #fff);
+      border: var(--terra-switch-slider-selected-active-border, 0);
+      box-shadow: var(--terra-switch-slider-selected-active-box-shadow, inset 0 0 0 1px #fff, inset 5px 2px 4px 2px #e6e7e8, 2px 0 2px 0 rgba(0, 0, 0, 0.2),
+      -2px 0 2px 0 rgba(0, 0, 0, 0.2), -1px 0 0 0 #01639e, 0 0 4px 1px rgba(0, 0, 0, 0.2)); // pressed state.
+    }
+
+    &:focus[data-focus-styles-enabled='true']:not(.is-disabled) {
+      background-color: var(--terra-switch-slider-selected-focus-background-color, #fff);
+      border: var(--terra-switch-slider-selected-focus-border, 0);
+    }
+  }
+  
+  .tray {
+    border: var(--terra-switch-tray-border, 1px solid #909496);
+    border-radius: var(--terra-switch-tray-border-radius, 1.42857rem);
+    bottom: 0;
+    cursor: pointer;
```

Comment:
Does the button also need a cursor specified? When hovering on the button now, the pointer styles aren't being applied.

Reply:
[pointer](https://github.com/cerner/terra-core/blob/48fce2d9ca73773ac6f9891feaa7f2a56b36f864/packages/terra-switch/src/Switch.module.scss#L132) has been added for button.